test/specs/modules/Dropdown/Dropdown-test.js

```diff
@@ -474,6 +474,28 @@ describe('Dropdown', () => {
 
       instance.setSelectedIndex.should.not.have.been.calledOnce()
     })
+
+    it('will only set the selectedIndex to the first element if forceSelection is set to true (default)', () => {
```

Comment:
I added unit test, but i'm not understanding why it's failing.

it's complaining that does not have a 'selectedIndex' state. However, I know that this isn't true. I also followed the pattern in line 1987 which has the same line of code. It's probably something i'm missing with how the test should be written.

Logic:

Create the dropdown component
initialize the initial state with setState
call setProps to trigger a change which will call setSelectedIndex().
validate the state of selectedIndex
I'm not familiar enough with the test infrastructure to debug the test. Any insights on how to configure debugging would be helpful.

Reply:
@darewreck54 I've changed test quite a bit, now it passes.

> I'm not familiar enough with the test infrastructure to debug the test. Any insights on how to configure debugging would be helpful. All my attempts to get work debugging tools were in vain 😭  I don't have a normal solution for you there. I remove all non-related tests and debug them with `console`.